BasementRenovator.py

```diff
@@ -809,6 +809,28 @@ def paintEvent(self, event):
                 int(Qt.AlignRight | Qt.AlignBottom),
                 f"Base HP : {e.entity.config.baseHP}",
             )
+            textY += 16
+
+            if e.entity.config.stageHP is not None and e.entity.config.stageHP != "0":
+                painter.drawText(
+                    r.right() - 34 - 200,
+                    textY,
+                    200,
+                    12,
+                    int(Qt.AlignRight | Qt.AlignBottom),
+                    f"Stage HP : {e.entity.config.stageHP}",
```

Comment:
it might be cool to also display the calculated stage hp for the current file's stage; lmk if you need any help finding that data

Reply:
It seems like stage-related variables are stored in something usually referred as "floorInfo" but I'm not sure where this would be accessed from any given part of the file

Reply:
Here's how `testMapCommon` gets it:
```py
        floorInfo = (
            xmlLookups.stages.lookup(path=mainWindow.path)
            or xmlLookups.stages.lookup(name="Basement")
        )[-1]
```

Reply:
Ok, it's done and implemented
![image](https://user-images.githubusercontent.com/78291137/230735667-f4e88e23-28ae-4a00-965c-2b98602d25eb.png)